pkg/test/framework/components/stackdriver/stackdriver.yaml

```diff
@@ -41,7 +41,7 @@ spec:
         app: stackdriver
     spec:
       containers:
-      - image: gcr.io/istio-testing/fake-stackdriver:2.0
```

Comment:
Don't think this is really needed as 1.6 tests doesn't have tracing or edges test: https://github.com/istio/istio/blob/release-1.6/tests/integration/telemetry/stackdriver/stackdriver_filter_test.go

I have no problem in updating either though..